analysis.R

```diff
@@ -0,0 +1,92 @@
+  geom_line(data = all_data_infections_day, aes(x = Day, y = Sum, color = Country)) + 
+  xlab('Day of Year') + 
+  ylab('Number of Infections Per Day') + 
+  ggtitle('Infections Per Day by Country')
+
+# Based on the data and the line plot of infections per day, it seems as if Country X experienced
+# the first outbreak of the disease. Looking at the graph it can be seen that Country X saw its first
+# infections on day 120 of the year whereas, infections didn't begin in Country Y until around day 137 according
+# to the line plot. This graph seems to confirm that the disease outbreak began in Country X.
+
+
+#2. If Country Y develops a vaccine for the disease, is it likely to work for citizens of Country X?
+
+# plot distributions of markers 
+
+marker_sum_country_X <- data.frame(colSums(countryX_data[,c(3:12)])) #sum down marker columns
+marker_sum_country_Y <- data.frame(colSums(countryY_data[,c(3:12)])) #sum down marker columns
+
+marker_sum_country_X <- cbind(rownames(marker_sum_country_X), data.frame(marker_sum_country_X, row.names=NULL)) #make marker names row names
+marker_sum_country_Y <- cbind(rownames(marker_sum_country_Y), data.frame(marker_sum_country_Y, row.names=NULL)) #make marker names row names
+
+colnames(marker_sum_country_X) <- c('marker','instances') #change column names
+colnames(marker_sum_country_Y) <- c('marker','instances') #change column names
+
+marker_sum_country_X$instances <- marker_sum_country_X$instances/sum(marker_sum_country_X$instances) #normalize instances of markers to 1
+marker_sum_country_Y$instances <- marker_sum_country_Y$instances/sum(marker_sum_country_Y$instances) #normalize instances of markers to 1
+
+ggplot() + #graph barplot of different marker prevalence to determine different distribution of markers in populations
+  geom_bar(marker_sum_country_X, stat = 'identity', position = 'dodge', mapping = aes(x=marker, y=instances, fill = 'Country X'), alpha = 0.8) +
+  geom_bar(marker_sum_country_Y, stat = 'identity', position = 'dodge', mapping = aes(x=marker, y=instances, fill = 'Country Y'), alpha = 0.8) +
+  xlab('Marker') + 
+  ylab('Proportion of Infections by Marker') +
+  ggtitle('Distribution of Infections by Marker For Each Country')
+
+# Based on the comparison of the distributions of positive markers in each of the two countries
+# it seems as if Country Y developed an effective vaccine for its populous, then the vaccine
+# would most likely not be effective for the citizens of Country X. This is because most of the infections
+# in Country X show positive presence of Markers 1-5 and almost never show presence of markers 6-10.
+# However, in Country Y, the most prevalent markers are markers 6-10 with just a little prevalence of 
+# markers 1-5. So based on the countries having different distributions of the prevalence of markers,
+# it would seem to be that a vaccine developed for Country Y would not be very effective for the citizens
+# of Country X.
+
```

Comment:
Good commenting and efficient code that uses coding concepts that we covered in class – 4 points